src/index.js

```diff
@@ -1,26 +1,37 @@
-import React from "react";
+import React, { Component, PropTypes } from "react";
```

Comment:
Oh wow. Totally misread the line and thought that you had removed the React in addition to adding the destructuring. Coolio :+1: